app/src/main/scala/com/coralogix/operator/Main.scala

```diff
@@ -72,6 +72,7 @@ object Main extends App {
       .ensuring {
         console.putStrLnErr("Shutting down")
       }
+      .untraced
```

Comment:
I think you need to disable that on the Platform to actually disable it. Otherwise causes that get logged in inner ZIO programs still contain a trace.